ssh/ssh.go

```diff
@@ -0,0 +1,85 @@
+package ssh
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"strings"
+
+	"golang.org/x/crypto/ssh"
+)
+
+// SSHKey represents an SSH private/public key pair for authentication.
+type SSHKey struct {
```

Comment:
Probably just `Key` because package name is already `ssh`.
So `ssh.Key` is enough.